.github/workflows/checks.yml

```diff
@@ -8,6 +8,7 @@ permissions:
 env:
```

Comment:
How do you feel about adding a job to run `newrelic-admin validate-config newrelic.ini`?